### DISCUSSION TITLE
Rails Trix Editor

### DISCUSSION THREAD

**Discussion by leewynne** [forem/forem#18059] [Feature Requests]
Is it a viable option to enable the rails trix editor for forems that aren't tech/dev focused?

**Comment by maestromac** [comment]
Thank you for this suggestion @leewynne !

**Reply by leewynne** [reply]
Really? Is this a possibility? 😀

**Reply by maestromac** [reply]
No promises 😅

**Reply by leewynne** [reply]
It could be our little secret, all hidden behind a feature flag 😂
